packages/react/src/lib/AutoControlledComponent.tsx

```diff
@@ -187,8 +187,9 @@ export default class AutoControlledComponent<P = {}, S = {}> extends UIComponent
    * Second argument is a state object that is always passed to setState.
    * @param {object} maybeState State that corresponds to controlled props.
    * @param {object} [state] Actual state, useful when you also need to setState.
+   * @param {object} onStateChanged Callback which is called after setState applied.
    */
-  trySetState = (maybeState, state?) => {
+  trySetState = (maybeState, state?, onStateChanged?: () => void) => {
```

Comment:
https://github.com/Semantic-Org/Semantic-UI-React/pull/3452#discussion_r264972028

I support there Levi, I think that we can also remove `state` param, too

Reply:
I like that, actually didn't find any usage of 2nd param.
Would make this change if nobody minds